src/main/java/ch/uzh/ifi/fs22/sel/group15/cozycave/server/rest/dto/ListingPutDto.java

```diff
@@ -72,14 +72,14 @@ public ListingPutDto(UUID id, String name, String description,
         this.rent = rent;
         this.deposit = deposit;
         this.rooms = rooms;
-        //this.publisher = publisher;
+        this.publisher = publisher;
     }
 
     // Constructor for multiple Picture add
     public ListingPutDto(UUID id, String name, String description,
                          LocationDto address, boolean published, List<Picture> pictures,
                          double sqm, ListingType listingtype, boolean furnished, List<Gender> availableTo,
-                         boolean available, double rent, double deposit, double rooms) {
+                         boolean available, double rent, double deposit, double rooms, UserGetDto publisher) {
```

Comment:
I thought you just wanted the frontend to send the uuid of the user, not the whole user. But I would even think of dropping this and setting is by the authenticated and executing user. So it can not be manipulated